<?xml version="1.0"?>
 
<launch>
    <arg name="maps_dir"       default="test/multi_maps/nakaniwa"/>
    <arg name="waypoints_file" default="test/multi_maps/nakaniwa/waypoints.yaml"/>
    <arg name="start_map_num"  default="0"/>


    <node name="map_server" pkg="map_server" type="map_server" args="$(arg maps_dir)/map$(arg start_map_num).yaml"/>

    <node pkg="amcl" type="amcl" name="amcl" output="screen"/>


    <node pkg="multi_map_manager" type="map_changer" name="map_changer" output="screen">
        <param name="multi_map_dir"     value="$(arg maps_dir)"/>
        <param name="waypoints_file"    value="$(arg waypoints_file)"/>
    </node>
 
</launch>